array/array.mbti

```diff
@@ -8,6 +9,7 @@ alias @moonbitlang/core/quickcheck as @quickcheck
 impl FixedArray {
   all[T](Self[T], (T) -> Bool) -> Bool
   any[T](Self[T], (T) -> Bool) -> Bool
+  blit_from_bytesview(Self[Byte], Int, @bytes.BytesView) -> Unit
```

Comment:
This method was in `builtin/bytesview.mbt`. It doesn't have to be in builtin so I moved it here when I deleted `builtin/bytesview.mbt`